src/Route.php

```diff
@@ -252,6 +276,11 @@ public function process(ServerRequestInterface $request, RequestHandlerInterface
     {
         if ($this->stack === null) {
             for ($i = count($this->middlewares) - 1; $i >= 0; $i--) {
+                if (is_string($this->middlewares[$i])) {
+                    $this->middlewares[$i] = $this->container->get($this->middlewares[$i]);
+                } elseif (is_array($this->middlewares[$i])) {
```

Comment:
How about to check indexes? :)

Reply:
It was checked in the prepareMiddleware()